daq/combine_reports_from_date_range.py

```diff
@@ -0,0 +1,106 @@
+import os
```

Comment:
module name is too specific, what's the next abstraction up?   report_combinator?  Also, should be a noun not a verb (combinator not combine)

Reply:
This is just meant to be a script, not a module so I think it makes sense to use an action for the filename. Just like setup_daq.